framework/uifactory/window.js

```diff
@@ -250,12 +251,37 @@ module.exports = function(args) {
 			$this.addActivityButton(opt);
 		};
 
+		/**
+		 * **Android specific**
+		 * Set the action buttons for this window
+		 * @method  setActivityButtons
+		 * @param {Array} buttons An Array of buttons activity buttons
+		 */
+		$this.setActivityButtons = function(buttons) {
+			activityButtons = [];
+			$this.menuItems = {};
```

Comment:
You set $this.menuItems to an empty object but don't populate it in the function. Is this done by `invalidateOptionsMenu`?

Reply:
@Jei